exercises/practice/sublist/.meta/config.json

```diff
@@ -23,17 +23,16 @@
   ],
   "files": {
     "solution": [
-      "src/main/java/RelationshipComputer.java"
+      "src/main/java/RelationshipComputer.java",
+      "src/main/java/Relationship.java"
     ],
     "test": [
       "src/test/java/RelationshipComputerTest.java"
     ],
     "example": [
       ".meta/src/reference/java/RelationshipComputer.java"
     ],
-    "editor": [
-      "src/main/java/Relationship.java"
-    ]
+    "editor": []
```

Comment:
When the array is empty, the key should be removed entirely. If you want, you can use [configlet](https://github.com/exercism/configlet) locally to check and auto-fix these issues.

Reply:
Sure.. doing it..